src/main/java/com/moz/qless/Queue.java

```diff
@@ -57,10 +61,11 @@ public int getHeartbeat() throws IOException {
     return Integer.parseInt(heartbeat.toString());
   }
 
-  public int getMaxConcurrency() throws IOException {
-    return Integer.parseInt(
-        this.client.getConfig().get(LuaConfigParameter.MAX_CONCURRENCY)
-        .toString());
+  public Integer getMaxConcurrency() throws IOException {
```

Comment:
Yeah, I agree this is kind of weird. A queue doesn't have to have a max concurrency set, so I wanted to use `null` to signal that case. In that case, it's assumed to be infinite. Interestingly enough, the Ruby bindings don't seem to provide a way to unset the max concurrency. I imagine that in practice, it's just set to an arbitrarily big number to 'unset' it.

Should I just change `setMaxConcurrency` to accept an `Integer`?